src/helpers/indexing-helper.php

```diff
@@ -161,6 +160,8 @@ public function set_reason( $reason ) {
 		$this->notification_center->remove_notification_by_id(
 			Indexing_Notification_Integration::NOTIFICATION_ID
 		);
+
+		$this->set_started( null );
```

Comment:
dit werkt prima. ik ben er alleen niet zo gelukkig mee dat je op deze plek _precies_ moet weten wat je moet doen zodat een andere method de juiste boolean teruggeeft. 
Als er ook maar iets wijzigt, en dat gaat vast gebeuren, is er een groot risicio op onverwacht gedrag.

Reply:
hier: (of r155 )
```suggestion
		$this->options_helper->set( 'indexing_reason_changed', $now );
```

@should_show_notification:
```if ( indexing_reason_changed> indexing_started ) { return true; }```

Reply:
Nice improvements.